scripts/run_integration_topology_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Script to locally run the integration topology test.
+#
+
+TEST_RUNNER="./bazel-bin/integration_test/src/python/topology_test_runner/topology-test-runner.pex"
+
+JAVA_TESTS_DIR="integration_test/src/java/org/apache/heron/integration_topology_test/topology"
+PYTHON_TESTS_DIR="integration_test/src/python/integration_test/topology"
+SCALA_TESTS_DIR="integration_test/src/scala/org/apache/heron/integration_test/topology"
+
+# integration test binaries have to be specified as absolute path
+JAVA_INTEGRATION_TESTS_BIN="${PWD}/bazel-genfiles/integration_test/src/java/integration-topology-tests.jar"
+PYTHON_INTEGRATION_TESTS_BIN="${PWD}/bazel-bin/integration_test/src/python/integration_test/topology/heron_integ_topology.pex"
+SCALA_INTEGRATION_TESTS_BIN="${PWD}/bazel-genfiles/integration_test/src/scala/scala-integration-tests.jar"
+
+CORE_PKG="file://${PWD}/bazel-bin/scripts/packages/heron-core.tar.gz"
+
+set -e
+
+# building tar packages
+DIR=`dirname $0`
+source ${DIR}/detect_os_type.sh
+bazel run --config=`platform` -- scripts/packages:heron-install.sh --user
+bazel build --config=`platform` {heron/...,scripts/packages:tarpkgs,integration_test/src/...}
+
+# run the simple http server
+#${HTTP_SERVER} 8080 &
```

Comment:
any specific reason we keep the HTTP server, scala test,  python test commented out?

Reply:
Http server will be used in instance state validation in stateful processing, and scala and python tests may be added in the future.